opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -826,6 +847,7 @@ def collect(
 
                 start_time_unix_nano = self._previous_start_time_unix_nano
                 sum_ = current_sum + self._previous_sum
+                zero_count = current_zero_count + self._previous_zero_count
```

Comment:
`zero_count` should be propagated across collections in case of cumulative aggregation instead of resetting it every time